autoload/slime.vim

```diff
@@ -13,12 +14,12 @@ function! s:TargetConfig() abort
   if exists("b:slime_config")
     return b:slime_config
   end
-  let output = system(g:slime_target_config)
```

Comment:
so this … I realized isn't going to work 😐 

my original idea was to delegate configuration to a shell program, but if we use `system`, there's no STDIN to type into and STDOUT gets captured…

this needs to be vimscript code to query the user

Reply:
Agreed! The reason I chose to use a reference to a function are:
1. For sending data, you cannot always use a shell script: e.g. for vim-terminal,
2. For the configuration part, I thought that maybe one would want to use something more sophisticated, such as https://github.com/nvim-telescope/telescope.nvim,
3. If you really want to use a `system` call, you can easily embed that in a function.

Reply:
all true 👍